ref/general/deploying-openshift.adoc

```diff
@@ -0,0 +1,52 @@
+After you write your microservice applications, you can containerize and deploy them to OpenShift to orchestrate and automate your containers.
+
+== What is OpenShift?
+OpenShift is an open source application platform that is based on link:https://kubernetes.io/[Kubernetes].
+Kubernetes is an orchestration platform for containerized applications.
+After you develop and containerize your microservices, you might have many containers that must communicate with each other and scale as each service is needed.
+Kubernetes provides features that automate and manage containers to meet the needs of microservice developers and ops teams.
+
+OpenShift is considered a distribution of Kubernetes that takes the upstream Kubernetes project and adds fixes and helpful tooling.
+It is also thoroughly tested before every release.
+These additions support enterprise applications, and you can have peace of mind when you know that your application platform was thoroughly tested and debugged.
+
+== Running applications on OpenShift
+One of the benefits of running your applications on OpenShift is that you can deploy them to any cloud platform, or to your current on-premises structure.
+You can use the link:https://docs.openshift.com/online/starter/cli_reference/openshift_cli/getting-started-cli.html[OpenShift CLI] or link:https://github.com/openshift/odo[OpenShift command line tools] to develop your application.
+Then, containerize your application in Open Liberty containers and deploy them to an OpenShift cluster.
+
+For a step-by-step tutorial on deploying to OpenShift, see the link:https://openliberty.io/guides/cloud-openshift.html[Deploying microservices to OpenShift guide]. For more information about different OpenShift deployment options, see the link:https://www.openshift.com/learn/topics/deploy[OpenShift documentation].
+
+== Open Liberty Operator
+Kubernetes Operators implement the link:https://github.com/operator-framework[Operator framework] and can be used on Kubernetes or OpenShift.
+Operators are extensions to Kubernetes that are customized to automate tasks beyond the initial automation that Kubernetes or OpenShift provide.
+For more information about Operators, see the link:https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Kubernetes documentation].
+
+The Open Liberty Operator is a Level 5 Operator, which means that it has the highest level of enterprise capabilities.
+It automates updates of binding information among microservices, meaning that it connects microservices and maintains information about whether a particular microservice produces or consumes a service.
+The Open Liberty Operator is available for installation from link:https://operatorhub.io/operator/open-liberty[OperatorHub].
+You can also find it available as a Red Hat certified Operator from link:https://access.redhat.com/containers/#/registry.connect.redhat.com/ibm/open-liberty-operator[OpenShift Container Platform (OCP)].
+
+== See also
```

Comment:
should the guide be under a `Where to next?` heading? I had one topic that has both where to & see also. Not sure if that's how we want to do it but though.